v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,131 @@ title: Cluster Overview Page
+Node Status | <ul><li>The number of `LIVE` nodes in the cluster.</li><li>The number of `SUSPECT` nodes in the cluster. A node is considered suspect if its [liveness status is unavailable](cluster-setup-troubleshooting.html#node-liveness-issues) or the node is in the process of [decommissioning](#decommissioned-nodes).</li><li>The number of `DEAD` nodes in the cluster.</li>
 Replication Status | <ul><li>The total number of [ranges](architecture/overview.html#glossary) in the cluster.</li><li>The number of [under-replicated ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li><li>The number of [unavailable ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li>
 
+### Capacity metrics
+
+The Cluster Overview, Node List, and Node Map display **Capacity Usage** by the CockroachDB [store](architecture/storage-layer.html) (the directory on each node where CockroachDB reads and writes its data) as a percentage of the disk space that is **usable** on the cluster, locality, or node.
+
+Usable disk space is constrained by the following:
+
+- The maximum store size, which may be specified using the [`--store`](cockroach-start.html#store) flag when starting nodes. If no store size has been explicitly set, the actual disk capacity is used as the limit. This value is displayed on the Capacity graph in the [Storage dashboard](admin-ui-storage-dashboard.html#capacity).
+- Any disk space occupied by non-CockroachDB data. This may include the operating system and other system files, as well as the Cockroach binary itself.
+
+The Admin UI thus calculates **usable** disk space as the sum of empty disk space, up to the value of the maximum store size, and disk space that is already being **used** by CockroachDB data.
+
+{{site.data.alerts.callout_info}}
+{% include {{ page.version.version }}/misc/available-capacity-metric.md %}
+{{site.data.alerts.end}}
+
 ## Node List
 
-The **Node List** is the default view on the **Overview** page.
-<img src="{{ 'images/v20.1/admin-ui-node-list.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+The **Node List** groups nodes by locality. The lowest-level locality tier is used to organize the Node List. Hover over a locality to see all localities for the group of nodes.
+
+{{site.data.alerts.callout_success}}
+We recommend [defining `--locality` flags when starting nodes](cockroach-start.html#locality). CockroachDB uses locality to distribute replicas and mitigate [network latency](admin-ui-network-latency-page.html). Locality is also a prerequisite for enabling the [Node Map](#node-map-enterprise).
```

Comment:
I learned from working on some other thing that a CRDB node does not actually know anything about its geographic location.  It just does string-matching on the arguments passed via the CLI.  It's up to the user to make sure the string pairs they pass to `--locality` map to some approximation of the reality of where their `cockroach` binaries are running.

(I'm not asking you to do anything in this PR, just wanted to mention this.  I also filed #6997 because our docs on that flag do not make this clear, but we do mention it in another place.)